### PR TITLE
metrics: fix duplicated "id": 234 in tidb.json

### LINKED ISSUE #42562: Duplicated "id": 234 in tidb.json of Grafana Dashboard
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

Go to the "TiDB" page of Grafana and expand the "TopSQL" row.

### 2. What did you expect to see? (Required)
The "Ignore Event Per Minute" and "Stats Cache LRU Cost" panels should be displayed normally.

### 3. What did you see instead (Required)
The "Ignore Event Per Minute" and "Stats Cache LRU Cost" panels were not displayed normally, sometimes disappeared and sometimes there with expressions confused with each other.

### 4. What is your TiDB version? (Required)
v6.5.1

### PR DESCRIPTION
**Pull request by pingandb**
### What problem does this PR solve?

Issue Number: close #42562

Problem Summary:

### What is changed and how it works?
Duplicated id 234 of "Ignore Event Per Minute" and "Stats Cache LRU Cost" panels caused them to be displayed abnormally, sometimes disappeared and sometimes there with expressions confused with each other. The fix is to assign them different ids.

### Check List
The "Ignore Event Per Minute" and "Stats Cache LRU Cost" panels should be displayed normally with the fix.

Tests 

The "Ignore Event Per Minute" and "Stats Cache LRU Cost" panels should be displayed normally with the fix.

Side effects

None

Documentation

None

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
Fix the issue that "Ignore Event Per Minute" and "Stats Cache LRU Cost" panels might not be displayed normally.
```

### CODE REVIEW

File: metrics/grafana/tidb.json [hunk 1]
```diff
@@ -16516,7 +16516,7 @@
             "y": 17
           },
           "hiddenSeries": false,
-          "id": 234,
+          "id": 299,
```
Comment (you06) [inline metrics/grafana/tidb.json:16519]:
```suggestion
          "id": 305,
```
The allocated id is increases to 304 after #42468, use 305 instead.

Comment (pingandb) [reply metrics/grafana/tidb.json:16519]: I did consider the risk of conflict. Given 305 the current max, 299 is less than 305 and not chosen yet, isn't it more guaranteed to not be chosen by others?

Comment (you06) [reply metrics/grafana/tidb.json:16519]: But 299 is already occupied.

https://github.com/pingcap/tidb/blob/9e8498510610d843268631f48b03b1f6280ce15d/metrics/grafana/tidb.json#L17432

Comment (pingandb) [reply metrics/grafana/tidb.json:16519]: Thanks for the comments. Will use 305.

Comment (pingandb) [reply metrics/grafana/tidb.json]: Change done.

**Review by you06 — APPROVED** [review]

**Comment by you06** [comment]
@lance6716 PTAL

**Review by lance6716 — APPROVED** [review]

**Pull request merged**